drivers/pinctrl/Kconfig.b91

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2022 Telink Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+config PINCTRL_TELINK_B91
+	bool "Telink B91 pin controller driver"
+	depends on SOC_RISCV_TELINK_B91
+	default y
```

Comment:
please default based on dt_compat_status, you can check some other drivers.

Reply:
Done